src/script/check_commands.sh

```diff
@@ -13,7 +14,7 @@ while read cmd; do
     fi
 done < commands.txt
 
-if [ "$missing_test" == true ]; then
+if [[ "$missing_test" == true ]]; then
```

Comment:
```suggestion
if $missing_test ; then
```
so no need to change the interpreter to bash.